src/core/Engine.cpp

```diff
@@ -126,9 +126,9 @@ void Engine::destroy()
 
 
 
-bool Engine::ignorePluginBlacklist()
+bool Engine::enableBlockedPlugins()
 {
-	const char* envVar = getenv("LMMS_IGNORE_BLACKLIST");
+	const char* envVar = getenv("LMMS_BLOCKED_PLUGINS");
```

Comment:
`LMMS_ENABLE_BLOCKED_PLUGINS` would be more clear.

And should this be made specific to Lv2 or do we want to keep it plugins in general?

Reply:
Funny, I also thought that this might be unclear. I changed it to `LMMS_ENABLE_BLOCKED_PLUGINS`.

I put it into `Engine` to allow this to be re-used by other plugin formats. Until any problems arise, I would like to keep this shared between plugin formats.

Reply:
> I put it into `Engine` to allow this to be re-used by other plugin formats. Until any problems arise, I would like to keep this shared between plugin formats.

Just sharing my thoughts here, but I don't know how much I like the idea of `Engine` having to know about blocked plugins (I don't see the two being that related). I would think this would be somewhere else, possibly some class we have related to general plugin management.

Reply:
@sakertooth This is a good point. I was simply looking for any class where I can easily put it. Thinking about alternatives: What about `ConfigManager`?

Reply:
> This is a good point. I was simply looking for any class where I can easily put it. Thinking about alternatives: What about ConfigManager?

IMO, it seems reasonable for the blocking of plugins to be handled in `PluginFactory` (at least for now), since it's related to plugin discovery. Not sure if this is the most optimal place to put it, but it seems okay to me.

Reply:
I put it into `ConfigManager` now, because for me, it is something that could almost go into the `SettingsDialog`. Almost, because it might be better hidden from most users and might be better used by developers.